core/unit_test/cuda/TestCuda_DebugPinUVMSpace.cpp

```diff
@@ -34,8 +34,8 @@ struct CopyFunctor {
 };
 
 TEST(cuda, debug_pin_um_to_host) {
-#ifdef KOKKOS_ARCH_AMPERE87
-  GTEST_SKIP() << "skipping for Jetson devices that have integrated memory";
+#if defined(KOKKOS_ARCH_AMPERE87) || defined(KOKKOS_ARCH_HOPPER90)
```

Comment:
should we only skip for debug builds?

Reply:
I decided to just revert for now since I can't reproduce locally and it fails a lot less frequently than the other builds but I have seen failures both in Debug and Release mode.